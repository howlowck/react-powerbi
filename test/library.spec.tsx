import React from 'react'
import { shallow } from 'enzyme'
import PowerbiEmbedded from '../src/index'

// eslint-disable-next-line no-undef
test('renders a component', () => {
  const embedded = shallow(<PowerbiEmbedded />)

  expect(embedded).toMatchSnapshot()
})
