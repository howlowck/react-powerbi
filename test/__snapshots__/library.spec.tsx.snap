// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a component 1`] = `
<div
  className="powerbi-frame"
  style={
    Object {
      "height": undefined,
      "width": undefined,
    }
  }
/>
`;
